src/utils.ts

```diff
@@ -47,3 +47,70 @@ export const subscribeKey = <T extends object>(
     }
   })
 }
+
+/**
+ * devtools
+ *
+ * This is to connect with [Redux DevTools Extension](https://github.com/zalmoxisus/redux-devtools-extension).
+ * Limitation: Only plain objects/values are supported.
+ *
+ * @example
+ * import { devtools } from 'valtio/utils'
+ * const state = proxy({ count: 0, text: 'hello' })
+ * const unsub = devtools(state, 'state name')
+ */
+export const devtools = <T extends object>(proxyObject: T, name?: string) => {
+  let extension: any
+  try {
+    extension = (window as any).__REDUX_DEVTOOLS_EXTENSION__
+  } catch {}
+  if (!extension) {
+    if (
+      typeof process === 'object' &&
+      process.env.NODE_ENV === 'development' &&
+      typeof window !== 'undefined'
+    ) {
+      console.warn('Please install/enable Redux devtools extension')
```

Comment:
For better messages, Can we use `[Warning] Please....` here?

Reply:
I don't have strong opinions on this message. Thought it's redundant, but if you think it's better, I can add it for sure.